src/spring/azext_spring/_params.py

```diff
@@ -677,6 +677,12 @@ def prepare_logs_argument(c):
                    help='If true, only import public certificate part from key vault.', default=False)
         c.argument('public_certificate_file', options_list=['--public-certificate-file', '-f'],
                    help='A file path for the public certificate to be uploaded')
+        c.argument('enable_auto_sync', options_list=['--enable-auto-sync'], arg_type=get_three_state_flag(),
+                   help='Whether to automatically synchronize certificate from key vault', default=False)
+
+    with self.argument_context('spring certificate update') as c:
+        c.argument('enable_auto_sync', options_list=['--enable-auto-sync'], arg_type=get_three_state_flag(),
+                   help='Whether to automatically synchronize certificate from key vault')
```

Comment:
Please do not explicitly repeat the default option name
```suggestion
        c.argument('enable_auto_sync', arg_type=get_three_state_flag(),
                   help='Whether to automatically synchronize certificate from key vault', default=False)

    with self.argument_context('spring certificate update') as c:
        c.argument('enable_auto_sync', arg_type=get_three_state_flag(),
                   help='Whether to automatically synchronize certificate from key vault')
```

Reply:
updated